iquip/apps/logger.py

```diff
@@ -136,41 +147,74 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         # connect signals to slots
         self.loggerFrame.clearButton.clicked.connect(self.checkToClear)
         self.confirmFrame.confirmed.connect(self.clearLog)
-        self.handler = LoggingHandler(self.addLog)
-        # TODO(aijuh): Change the log format when it is determined.
-        fs = "%(levelname)s [%(name)s] [%(filename)s:%(lineno)d] %(message)s"
-        formatter = logging.Formatter(fs)
-        self.handler.setFormatter(formatter)
+        #initialize logger
+        self.initLogger()
+        # set loggerFrame's frameLevelBox
+        self.loggerFrame.frameLevelBox.addItems(self.levels_dict.values())
+        self.loggerFrame.frameLevelBox.textActivated.connect(self.setFrameLevel)
+        self.loggerFrame.frameLevelBox.setCurrentText(self.levels_dict[self.frameHandler.level])
+        # set loggerFrame's fileLevelBox
+        self.loggerFrame.fileLevelBox.addItems(self.levels_dict.values())
+        self.loggerFrame.fileLevelBox.textActivated.connect(self.setFileLevel)
+        self.loggerFrame.fileLevelBox.setCurrentText(self.levels_dict[self.fileHandler.level])
+
+    def initLogger(self):
+        """Initializes the root logger and handlers for constructor."""
+        self.levels_dict = {10: "DEBUG", 20: "INFO", 30: "WARNING", 40: "ERROR", 50: "CRITICAL"}
+        # initialize handlers
+        self.frameHandler = LoggingHandler(self.addLog)
+        self.fileHandler = handlers.TimedRotatingFileHandler(filename = "log", when = "midnight",
+                                                             interval = 1, encoding = 'utf-8')
+        self.fileHandler.suffix = '-%Y%m%d'
+        simpleFormat = "[%(name)s] %(message)s"
+        complexFormat = "%(asctime)s %(levelname)s [%(name)s]"\
+                        " [%(filename)s:%(lineno)d] %(message)s"
+        self.frameHandler.setFormatter(logging.Formatter(simpleFormat))
+        self.fileHandler.setFormatter(logging.Formatter(complexFormat))
+        # set rootLogger
         rootLogger = logging.getLogger()
-        rootLogger.addHandler(self.handler)
-        self.setLevel("WARNING")
-        self.loggerFrame.levelBox.addItems(["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"])
-        self.loggerFrame.levelBox.textActivated.connect(self.setLevel)
-        self.loggerFrame.levelBox.setCurrentText("WARNING")
+        rootLogger.addHandler(self.frameHandler)
+        rootLogger.addHandler(self.fileHandler)
+        self.fileHandler.setLevel("WARNING")
```

Comment:
I think you should set the `rootLogger` level here since you set the others below.